src/query/api/v1/handler/graphite/find_parser.go

```diff
@@ -49,46 +57,64 @@ func parseFindParamsToQuery(r *http.Request) (
 		untilString = "now"
 	}
 
-	from, err := graphite.ParseTime(
+	end, err := graphite.ParseTime(
```

Comment:
Shouldn't this be start? And the other be end?

Reply:
nit: maybe just leave it as `from` and `until` to be consistent with the URL parameter names.

Reply:
Good catch; added it to the test.